### DISCUSSION TITLE
C# Detected as Shaderlab - Unity Projects

### DISCUSSION THREAD

**Discussion by TegridyLtd** [github-linguist/linguist#5654] [Classification] [answered]
https://github.com/TegridyLtd/TegridyBoatController
https://github.com/TegridyLtd/TegridyLanguageController
https://github.com/TegridyLtd/TegridyLander
https://github.com/TegridyLtd/TegridyPuzzleMatch
https://github.com/TegridyLtd/TegridyPuzzleMatch
https://github.com/TegridyLtd/TegridyDayController
https://github.com/TegridyLtd/TegridyBandit

Do I need to set an override my self or is it done by someone in the shadows?

**Comment by lildude** [comment]
You do it.

**Comment by TegridyLtd** [comment]
> 

Any hints how? new to ghithub

**Reply by lildude** [reply, marked as answer]
Second point of the documentation links in the [README.md](https://github.com/github/linguist#documentation) 😉
